Blizzard.IO.Core/IConsumer.cs

```diff
@@ -0,0 +1,11 @@
+﻿using System;
+
+namespace Blizzard.IO.Core
+{
+    public interface IConsumer<TData>
+    {
+        void StartConsume();
+        void StopConsume();
+        event Action<TData> MessageConsumed;
```

Comment:
I believe it should be `MessageReceived`

Reply:
So the class name is should be Receiver or MessageReceiver?